blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,121 @@
+
+Making configuration changes through the JSON editor is now easier too with easy to access documentation in the editor itself.
+
+<!-- TODO (ryan): GIF showing editor using the help tool tip with an item that requires a choice of values -->
+
+## Improved configuration management and documentation
+
+Sourcegraph 3.0 introduced the [management console](https://about.sourcegraph.com/blog/sourcegraph-3.0#management-console-and-site-config-improvements) for editing critical configuration to allow recovery from misconfiguration in the site admin.
+
+We've [updated our documentation](https://docs.sourcegraph.com/admin/config) to reflect this change, separating configuration into two sections:
+
+- [Critical configuration](https://docs.sourcegraph.com/admin/config/critical_config).
+- [Site configuration](https://docs.sourcegraph.com/admin/config/site_config).
+
+The documentation pages for [site configuration](https://docs.sourcegraph.com/admin/config/site_config#reference) and [critical configuration](https://docs.sourcegraph.com/admin/config/critical_config#reference) now describe each option and give examples, to make it easier for you to configure your Sourcegraph instance. You can get the same documentation and examples in the JSON editors for both types of configuration, too.
+
+## New Sourcegraph extension APIs
+
+The [Sourcegraph extension API](https://docs.sourcegraph.com/extensions) has two new APIs to make it easier to add interactive features to code views on Sourcegraph, GitHub, and other similar tools:
+
+- [API for accessing the current text selection range(s)](https://github.com/sourcegraph/sourcegraph/issues/702)
+- [APIs for listening to changes to the active text editor and its selections](https://github.com/sourcegraph/sourcegraph/issues/2026).
+
+This means the [Git extras Sourcegraph extension](https://sourcegraph.com/extensions?query=git-extras) works better than ever. Want this functionality on GitHub too? Just install the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension).
+
+<!-- TODO (ryan): Gif of git-extras extension -->
+
+## Fixed release schedule: a new Sourcegraph release on the 20th of every month
+
+Since shipping version 2.4 in January 2018, we've released 11 versions of Sourcegraph. Shipping regularly means progress is constant and predictable.
+
+We're constantly looking for ways to improve our release process. Inspired by [GitLab's fixed monthly release cycle](https://about.gitlab.com/2018/11/21/why-gitlab-uses-a-monthly-release-cycle/), Sourcegraph is now [released on the 20th of each month](https://docs.sourcegraph.com/dev/releases).
+
+We encourage feedback as early as possible so we can incorporate it into our product planning. View the [product roadmap](https://docs.sourcegraph.com/dev/roadmap) to track what features are slated for the next release.
+
+## Changelog
```

Comment:
Why not just link to the relevant section in the changelog?

Reply:
I see Quinn suggested otherwise. Here is my response: https://github.com/sourcegraph/about/pull/83#discussion_r258326794